libminifi/include/Connection.h

```diff
@@ -58,8 +58,8 @@ class Connection : public core::Connectable {
   Connection(const Connection &parent) = delete;
   Connection &operator=(const Connection &parent) = delete;
 
-  static constexpr uint64_t DEFAULT_BACKPRESSURE_SIZE_THRESHOLD = 10000;
-  static constexpr uint64_t DEFAULT_BACKPRESSURE_DATA_THRESHOLD = 10_MB;
+  static constexpr uint64_t DEFAULT_BACKPRESSURE_THRESHOLD_COUNT = 10000;
+  static constexpr uint64_t DEFAULT_BACKPRESSURE_THRESHOLD_DATA_SIZE = 10_MB;
```

Comment:
I think we can cut the count in half, but increase the data size at least 10x, to 100 MB. Those defaults would be more reasonable in most cases IMO. 
```suggestion
  static constexpr uint64_t DEFAULT_BACKPRESSURE_THRESHOLD_COUNT = 5000;
  static constexpr uint64_t DEFAULT_BACKPRESSURE_THRESHOLD_DATA_SIZE = 100_MB;
```

Reply:
based on our conversation I've modified it to 2000 / 100MB in https://github.com/apache/nifi-minifi-cpp/pull/1501/commits/75e3d93ddd845f30c8674ca03b4f62b2bf0faf97